R/pkg/R/mllib_recommendation.R

```diff
@@ -82,6 +82,12 @@ setClass("ALSModel", representation(jobj = "jobj"))
 #' statsS <- summary(modelS)
 #' }
 #' @note spark.als since 2.1.0
+#' @note the input rating dataframe to the ALS implementation should not be nondeterministic.
```

Comment:
not be nondeterministic -> be deterministic
I might remove 'probably' below

Reply:
ok.